src/components/modals/RedeemModal.tsx

```diff
@@ -54,6 +60,7 @@ export default function RedeemModal({
     : rewardAmount
 
   function redeem() {
+    form.validateFields()
```

Comment:
good call validating here! though this actually returns a promise, so we'll want to use `await form.validateFields()`